clients/src/test/java/org/apache/kafka/common/requests/OffsetFetchRequestTest.java

```diff
@@ -44,25 +46,20 @@
     private final int partitionOne = 1;
     private final String topicTwo = "topic2";
     private final int partitionTwo = 2;
-    private final String groupId = "groupId";
+    private final String topicThree = "topic3";
+    private final String group1 = "group1";
+    private final String group2 = "group2";
+    private final String group3 = "group3";
+    private final String group4 = "group4";
+    private final String group5 = "group5";
```

Comment:
The reason I had individually defined each group here is because we have to put the groups individually in the map and associated with its specific topic partition. It also made it easier to verify responses after, since we get the partition responses on a group level. Do you have an example of how putting the groups in `List<String>` would help reduce the repetition?